api/envoy/config/filter/network/dubbo_proxy/v2alpha1/route.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.dubbo_proxy.v2alpha1;
+option java_package = "io.envoyproxy.envoy.config.filter.network.dubbo_proxy.v2alpha1";
+option java_multiple_files = true;
+option go_package = "v2";
+
+import "envoy/api/v2/route/route.proto";
+import "envoy/type/range.proto";
+
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+import "gogoproto/gogo.proto";
+
+// [#protodoc-title: Dubbo route configuration]
+
+message RouteConfiguration {
+  // The name of the route configuration. Reserved for future use in asynchronous route discovery.
+  string name = 1;
+
+  // The interface name of the service.
+  string interface = 2;
+
+  // Which group does the interface belong to.
+  string group = 3;
+
+  // The version number of the interface.
+  string version = 4;
+
+  // The list of routes that will be matched, in order, against incoming requests. The first route
+  // that matches will be used.
+  repeated Route routes = 5 [(gogoproto.nullable) = false];
+}
+
+message Route {
+  // Route matching prarameters.
+  RouteMatch match = 1 [(validate.rules).message.required = true, (gogoproto.nullable) = false];
+
+  // Route request to some upstream cluster.
+  RouteAction route = 2 [(validate.rules).message.required = true, (gogoproto.nullable) = false];
+}
+
+message MethodMatch {
+  // The name of the method.
+  string name = 1;
```

Comment:
I guess this is the exact match and you only want to support that? If so comment that it is an exact match.

Reply:
Sorry，method names support wildcard matching, and I added some comments.

Reply:
If so use StringMatch?

Reply:
StringMatcher does not support wildcards and is currently used sparely, so I'm not sure if it should be added to StringMatcher. the wildcards I need to support are * and ?.

Reply:
StringMatcher has prefix/suffix/regex so I think it is enough for your case. Or is that a part of Dubbo spec to specifying method with wildcard? I'm in favor of consistency of the config if it is not a spec. Shouldn't introduce a new matching semantics as we already have enough.

Reply:
agree, I've changed it.